pkg/client/client.go

```diff
@@ -164,12 +166,41 @@ func NewClient(p *properties.Properties, workload ycsb.Workload, db ycsb.DB) *Cl
 
 // Run runs the workload to the target DB, and blocks until all workers end.
 func (c *Client) Run(ctx context.Context) {
+	var wg sync.WaitGroup
 	threadCount := c.p.GetInt(prop.ThreadCount, 100)
 
-	c.wg.Add(threadCount)
+	wg.Add(threadCount)
+	measureCtx, measureCancel := context.WithCancel(ctx)
+	go func() {
+		// load stage no need to warm up
+		if c.p.GetBool(prop.DoTransactions, true) {
+			dur := c.p.GetInt64(prop.WarmUpTime, 0)
+			select {
+			case <-ctx.Done():
+				return
+			case <-time.After(time.Duration(dur) * time.Second):
+			}
+		}
+		// finish warming up
+		measurement.FinishWarmUp()
```

Comment:
if we don't enable Warmup, but the default WarmUp is true, so when executing this, the following thread executions may run for a time, but they previous executed must be measured.